packages/relay/src/lib/types/debug.ts

```diff
@@ -13,6 +13,13 @@ export interface BlockTracerConfig {
   tracerConfig?: ICallTracerConfig;
 }
 
+export interface TransactionTracerConfig {
+  /** The type of tracer to use for block tracing. */
+  tracer: TracerType;
+  /** Optional configuration for the call tracer. */
+  tracerConfig?: ICallTracerConfig | IOpcodeLoggerConfig;
```

Comment:
this looks like `type ITracerConfig`, I don't mind inlining (actually might make more sense here). But if `ITracerConfig` is not used or duplicated we should unify the types used to avoid confusion.